docs/index.rst

```diff
@@ -78,6 +78,7 @@ anonymous sources.
    yubikey_setup
    backup_and_restore
    backup_workstations
+   upgrade_tails_usbs
```

Comment:
Since the document title is "Updating Tails USBs", I would suggest naming the file update_tails_usbs (the terms are clearly often used interchangeably; the main reason to care about it here is to avoid confusion when locating the file).